content/luau/concepts/tables/tables.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'Tables'
+Description: 'In the context of luau and Roblox, tables are data structures that are capable of storing values of any type other than nil.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/luau'
+CatalogContent:
+  - 'paths/computer-science'
+Subjects:
+  - 'Developer Tools'
+  - 'Game Development'
+Tags:
+  - 'Markdown'
+  - 'GitHub'
+  - 'Documentation'
+---
+
+The table data type can store multiple values of any type that isn't nil, including booleans, numbers, strings, functions, and other tables.
+
+The syntax for an empty table is:
+
+```
+local table = {}
+```
+
+In the context of [Luau](https://www.codecademy.com/resources/docs/luau), the local keyword is primarily used to declare local variables. It is not used specifically for instantiating tables. You can use the local keyword to create local variables of various types, including tables, numbers, strings, functions, and more. What creates the table is the curly braces in the syntax ({}). If you want to instantiate a table with parameters you can write something like this:
+
+```
+local table = {"amy", "john", "bob"} 
+```
+
+The above example is instantiated like an array, but as mentioned above, table data types are capable of storing any form of data other than nil. Here is another example:
+
+```
+local profile = {"john",  24, "Atlanta",  false}
+```
+
+Not only can tables be used as arrays, but you can even use them as dictionaries. Here is an example of a table in the form of a dictionary with an array data type included:
+
+```
+local profile = {
+    username = "john_doe",
+    level = 12,
+    experience = 500,
+    items = {"sword", "shield", "knife", "helmet"},
+}
+```
+
+
+### Accessing Items
+
+
+If your table is set up like an array, an important thing to note is that the index begins at 1 rather than at 0 as it is for other languages. Let's use this table as an example:
```

Comment:
```suggestion
If your table is set up like an [array](https://www.codecademy.com/resources/docs/javascript/arrays), an important thing to note is that the index begins at 1 rather than at 0 as it is for other languages. Let's use this table as an example:
```

Reply:
Alright. I'm not seeing a path for dictionary documentation. Does that not exist yet? lol Should I just create a hyperlink for each datatype?

Reply:
I linked it in there. Here it is: https://www.codecademy.com/resources/docs/javascript/arrays

Reply:
Yeah, I included that one when you shared it, but I was going to include a link for my dictionary example because I thought you might mention it.

Reply:
nvm overlooked it in the python directory